front/client/components/Article.vue

```diff
@@ -60,30 +30,33 @@
 <script lang="ts">
 import Vue from 'vue'
 import Component from 'vue-class-component'
+import ArticleContent from 'client/components/ArticleContent.vue'
 
-
-@Component
+@Component({
+  components: {
+    ArticleContent
+  }
+})
 export default class ArticleItem extends Vue {
+  // description: boolean = false
+  categoryDescription: string = ''
 
   async asyncData() {
-    return this.$store.dispatch('categoryItem/loadCategoryItem')
+    return this.$store.dispatch('categoryItem/loadCategoryItem').then(() => {
+      this.categoryDescription = this.$store.state.categoryItem.categoryItemList.description.html
+    })
   }
 
   get getCategory () {
-    // why this returns items[]?
-    // return this.$store.state.categoryItem.categoryItemList
     return this.$store.state.categoryItem
   }
 
-  get getCategoryDescription() {
-    const description = this.$store.state.categoryItem.categoryItemList.description.html;
-    return description
-  }
-
   get getCategoryItems() {
-    const items = this.$store.state.categoryItem.categoryItemList.items
+    return this.$store.state.categoryItem.categoryItemList.items
+  }
 
-    return items
+  created() {
```

Comment:
Delete it. No reason to trigger asyncData function manually, it triggers automatically